src/Psr7/ControllerHelperTrait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace MPScholten\RequestParser\Psr7;
+
+use MPScholten\RequestParser\RequestParser;
+use MPScholten\RequestParser\TypeParser;
+use Psr\Http\Message\ServerRequestInterface;
+
+trait ControllerHelperTrait
```

Comment:
I wonder if we could perhaps achieve some level of abstraction for `ControllerHelperTrait`, given that it's both (Symfony's and Psr7's) are almost identical except for the `init...` part of them.